lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module

```diff
@@ -26,6 +26,37 @@ function dosomething_campaign_form_campaign_node_form_alter(&$form, &$form_state
   $form['#after_build'][] = 'dosomething_campaign_form_campaign_node_form_after_build';
 }
 
+function dosomething_campaign_init() {
+  // Get the URL and split it up
+  $current_path = current_path();
+  $path_items = explode('/', $current_path);
+
+  // Verify we're dealing with a node edit with no translation specification in the URL
+  if ($path_items[0] == "node" && is_numeric($path_items[1]) && count($path_items) == 3) {
```

Comment:
this catches all those node tabs at the top
`node/[id]/revisions`
`node/[id]/pitch`
`node/[id]/custom-settings`
`node/[id]/rb`
and redirects to the edit page